src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -116,6 +122,17 @@ class ContinuousBatchingPipeline::Impl {
         return m_pipeline_metrics;
     }
 
+    std::vector<std::string> get_model_configuration() {
+        return m_model_config_namevalues;
+    }
+
+    void print_model_configuration() {
+        std::cout << "Loaded model configuration:" << std::endl;
```

Comment:
That's not the moment when the configuration is loaded. It happens earlier. Either remove the word `Loaded` or move the print to where it actually happens.